src/main/java/seedu/address/logic/commands/AddPairCommand.java

```diff
@@ -17,11 +17,11 @@ public class AddPairCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Pairs an elderly and volunteer in the address book. "
             + "Parameters: "
-            + PREFIX_ELDERLY_NRIC + "ELDERLY ID "
-            + PREFIX_VOLUNTEER_NRIC + "VOLUNTEER ID "
+            + PREFIX_NRIC_ELDERLY + "ELDERLY ID "
+            + PREFIX_NRIC_VOLUNTEER + "VOLUNTEER ID "
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_ELDERLY_NRIC + "1234 "
-            + PREFIX_VOLUNTEER_NRIC + "5678 ";
+            + PREFIX_NRIC_ELDERLY + "1234 "
+            + PREFIX_NRIC_VOLUNTEER + "5678 ";
 
```

Comment:
Am I understanding correctly that we need to use elderly and volunteer's NRIC for pairing, so we need to separate their prefixes?

Reply:
Yup I believe that was the idea. We had this in the user guide as well, so I decided to follow it.